en/SetupView/safety_ardupilot.md

```diff
@@ -0,0 +1,202 @@
+The thresholds can be disabled by setting them to zero.
+
+![Safety Setup - Battery1 Failsafe Triggers (Copter)](../../assets/setup/safety/safety_arducopter_battery1_failsafe_triggers.jpg)
+
+The configuration options are:
+- **Low action** ([BATT_FS_LOW_ACT](http://ardupilot.org/copter/docs/parameters.html#batt-fs-low-act-low-battery-failsafe-action)) - Select one of: None, Land, RTL, SmartRTL, SmartRTL or Land, Terminate.
+- **Critical action** ([BATT_FS_CRT_ACT](http://ardupilot.org/copter/docs/parameters.html#batt-fs-crt-act-critical-battery-failsafe-action)) - Select one of: None, Land, RTL, SmartRTL, SmartRTL or Land, Terminate.
+- **Low voltage threshold** ([BATT_LOW_VOLT](http://ardupilot.org/copter/docs/parameters.html#batt-low-volt-low-battery-voltage)) - Battery voltage that triggers the *low action*. 
+- **Critical voltage threshold** ([BATT_CRT_VOLT](http://ardupilot.org/copter/docs/parameters.html#batt-crt-volt-critical-battery-voltage))- Battery voltage that triggers the *critical action*. 
+- **Low mAh threshold** ([BATT_LOW_MAH](http://ardupilot.org/copter/docs/parameters.html#batt-low-mah-low-battery-capacity)) - Battery capacity that triggers the *low action*. 
+- **Critical mAh threshold** ([BATT_CRT_MAH](http://ardupilot.org/copter/docs/parameters.html#batt-crt-mah-battery-critical-capacity)) - Battery capacity that triggers the *critical action*. 
+
+
+<!-- Questions
+- What about Battery 2?
+- What actions do BATT_LOW_MAH (capacity) trigger. The ones listed in params don't exist. Is it just same as voltage ones?)
+--> 
+
+### General Failsafe Triggers {#failsafe_triggers_copter}
+
+This panel enables the [GCS Failsafe](http://ardupilot.org/copter/docs/gcs-failsafe.html) 
+and enables/configures the throttle failsafe.
+
+![Safety Setup - General Failsafe Triggers (Copter)](../../assets/setup/safety/safety_arducopter_general_failsafe_triggers.jpg)
+
+The configuration options are:
+- **Ground Station failsafe** - Disabled, Enabled always RTL, Enabled Continue with Mission in Auto Mode, Enabled Always SmartRTL or RTL, Enabled Always SmartRTL or Land.
+- **Throttle failsafe** - Disabled, Always RTL, Continue with Mission in Auto Mode, Always land.
+- **PWM Threshold** ([FS_THR_VALUE](http://ardupilot.org/copter/docs/parameters.html#fs-thr-value-throttle-failsafe-value)) - PWM value below which throttle failsafe triggers.
+
+<!-- Question 
+- no radio failsafe?
+- Link to throttle failsafe? 
+-->
+
+
+### Geofence {#geofence_copter}
+
+This panel sets the parameters for the cylindrical [Simple Geofence](http://ardupilot.org/copter/docs/ac2_simple_geofence.html). 
+You can set whether the fence radius or height are enabled, the maximum values for causing a breach, and the action in the event of a breach.
+
+![Safety Setup - Geofence (Copter)](../../assets/setup/safety/safety_arducopter_geofence.jpg)
+
+The configuration options are:
+
+- **Circle GeoFence enabled** () - Enable the circular geofence.
```

Comment:
How does this work for enabling? Doesn't seem to be corresponding parameter.

Reply:
Circle and Altitude GeoFence are controlled by the FENCE_ENABLE parameter which these checkboxes set appropriately.

Reply:
Ah, I think you mean FENCE_TYPE (+ FENCE_ENABLE). I'll update appropriately.